src/index.html

```diff
@@ -4,19 +4,117 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,maximum-scale=1.0, minimum-scale=1.0"
```

Comment:
The `content` attribute has a line that exceeds 80 characters. Consider breaking it into multiple lines to improve readability.